docs/gui.rst

```diff
@@ -20,7 +20,7 @@ Create a window
     Create a window.
     If ``res`` is scalar, then width will be equal to height.
 
-    This creates a window whose width is 1024, height is 768:
+    This creates a window whose width is 1024 and height is 768:
```

Comment:
I'd say ''This creates a window of 1024 x 768'' or "This creates a window whose resolution is 1024 x 768"

Reply:
Sounds good, thank you! Btw, please feel free to use the `suggestion` feature (item 6 in https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) of github to make us work more efficiently.

Reply:
Sure thanks, sorry I'm not so used to the github workflow.